content/docs/api-reference/assets-api/index.md

```diff
@@ -13,4 +13,171 @@ weight: 101
 toc: true
 ---
 
+## Asset API Examples
+
+Create the [bearer_token](../../setup-and-administration/getting-access-tokens-using-client-secret) and store in a file in a secure local directory with 0600 permissions.
+
+Set the URL (for example):
+
+```bash
+export URL=https://synsation.1234-5678.nodes.archivist.jitsuin.io 
```

Comment:
This should be “https://app.rkvst.io” in all the examples

Reply:
All should be updated